lib/gen_fsm.ex

```diff
@@ -20,69 +85,82 @@ defmodule GenFSM do
     end
   end
 
+  @spec stop(server, reason :: term, timeout) :: :ok
   def stop(fsm, reason \\ :normal, timeout \\ :infinity) do
     :gen.stop(fsm, reason, timeout)
   end
 
+  @type fsm_ref :: pid | name
```

Comment:
I think it will allow for it - I checked the erlang docs and http://erlang.org/doc/man/gen_fsm.html#send_event-2 and http://erlang.org/doc/man/gen_fsm.html#stop-3 have FsmRef listed and allow for  Name | {Name,Node} | {global,GlobalName} | {via,Module,ViaName} | pid().  

What do you think?